packages/twenty-front/src/modules/ui/input/components/IconPicker.tsx

```diff
@@ -147,6 +155,16 @@ export const IconPicker = ({
     [matchingSearchIconKeys],
   );
 
+  const getIconForIconPicker = () => {
```

Comment:
same here, I don't think we need this logic, we already have the selectedIconKey